docs/components/integration-framework/introduction.md

```diff
@@ -9,44 +9,51 @@ Camunda offers an **integration framework** which allows you to integrate extern
+1. The business logic is implemented by a [Connector function](./connectors/custom-built-connectors/connector-sdk.md#runtime-logic).
 2. The user interface during modeling is provided using a [Connector template](./connectors/custom-built-connectors/connector-templates.md).
 
-You can use **Connectors** from two different perspectives:
+You can use different sets of Connectors, depending on the environment you are using:
 
-#### Out-of-the-box Connectors
+### Camunda Platform 8 SaaS
 
-You can use [**out-of-the-box Connectors**](./connectors/use-connectors.md) provided by Camunda. When using Camunda Platform 8 SaaS, the runtime is operated by Camunda and offered in the scope of the product. The **out-of-the-box Connectors** are a solution which allow you to solve [everyday use cases](./connectors/out-of-the-box-connectors/available-connectors-overview.md) in process automation without the need to implement a single line of code.
+When using Camunda Platform 8 SaaS, the runtime is operated by Camunda and offered in the scope of the product. In this environment, you can use the [out-of-the-box Connectors](./connectors/out-of-the-box-connectors/available-connectors-overview.md) provided by Camunda. These Connectors allow you to solve everyday use cases in process automation without the need to implement a single line of code.
 
-#### Custom-built Connectors
+### Camunda Platform 8 Self-Managed
 
-You can use the Camunda [**Connector SDK**](./connectors/custom-built-connectors/connector-sdk.md) in combination with [**Connector templates**](./connectors/custom-built-connectors/connector-templates.md) to develop custom **Connectors**.
+In a Self-Managed environment, you provide the execution environment for Connectors yourself. Thus, you also decide which Connectors are available in your system.
+The Camunda [Connector SDK](./connectors/custom-built-connectors/connector-sdk.md) allows you to run multiple Connectors using [runtime environments](./connectors/custom-built-connectors/connector-sdk.md#runtime-environments).
+You can configure your runtime environment to execute any set of Connectors available to you.
 
-These custom **Connectors** can then be used by process developers in your Self-Managed environment in the same way they would use [**out-of-the-box Connectors**](./connectors/use-connectors.md).
+You can use the [out-of-the-box Connectors](./connectors/out-of-the-box-connectors/available-connectors-overview.md) provided by Camunda. Every Connector links to a source code repository that enables you to run the Connector in a Self-Managed environment.
+Besides those provided Connectors, you can use the Camunda Connector SDK in combination with [Connector templates](./connectors/custom-built-connectors/connector-templates.md) to develop [custom Connectors](./connectors/custom-built-connectors/connector-sdk.md#creating-a-custom-connector).
```

Comment:
We need to add a short note here regarding licensing, I think.

Reply:
Done with [52ceaf2](https://github.com/camunda/camunda-platform-docs/pull/1229/commits/52ceaf21811d6079eadc05345339f31eabbcbe57)